tests/e2e/support/index.js

```diff
@@ -29,4 +31,6 @@ Cypress.Cookies.defaults({
 
 beforeEach(() => {
   cy.loadModeler();
+  dragFromSourceToDest(nodeTypes.startEvent, { x: 150, y: 150 });
+  cy.get('.paper-container').click();
```

Comment:
This removes focus from the start event, which puts the paper in the same state that it was before these changes were made.